api/models/Game.js

```diff
@@ -119,5 +119,14 @@ module.exports = {
     winner: {
       model: 'user',
     },
+    lock: {
+      type: 'string',
+      allowNull: true,
+    },
+    // Time game was last locked in millis since epoch
+    lockedAt: {
+      type: 'number',
+      columnType: 'integer',
```

Comment:
It has been a mistake to fail to set the `columnType` of our other numbers because the default to `float` at the physical db layer even though most of our number columns are database ids. I'm hopeful that this practice will clean up new columns going forward. It's worth writing an issue to clean that up for our other columns, app-wide and to write a db migration to make it so.